### ISSUE TITLE
First time setup does not install steam, stuck at 50% for awhile before moving on.

### ISSUE THREAD

**Issue by Sildurs-shaders** [LukeShortCloud/winesapOS#553]
Hey I tried to install winesap minimal 3.2.0 in both vmware and hyper-v. In hyper-v the archlinux - steamos variant does not boot, uuid device error. But the main issue is that in both vwmare and hyper-v steam won't install correctly, the shortcuts are being created, just steam itself is not installed. Any ideas?

EDIT:
Updating the repos like you posted in the release page and then installing steam manually works. Running the initial setup again after that fixes the rest.

```
sudo sed -i s'/\[holo\]/\[holo-rel\]/'g /etc/pacman.conf
sudo sed -i s'/\[jupiter\]/\[jupiter-rel\]/'g /etc/pacman.conf
sudo pacman -S -y -y
sudo pacman -S steam
reboot
```

**Comment by LukeShortCloud** [comment]
@Sildurs-shaders Thanks for opening this issue! I plan to get a 3.2.1 release out soon to address this. It includes the workaround you mentioned.

**Comment by Sildurs-shaders** [comment]
Awesome, updating the repos before running the initial setup might be enough to get steam to install without the need to do it manually.

**Comment by LukeShortCloud** [comment]
We have officially released winesapOS 3.2.1 to address this issue: https://github.com/LukeShortCloud/winesapOS/releases/tag/3.2.1
